tests/unit/test_blob.py

```diff
@@ -997,7 +998,7 @@ def test_download_to_file_with_failure(self):
 
         blob_name = "blob-name"
         media_link = "http://test.invalid"
-        client = mock.Mock(spec=[u"_http"])
+        client = Client()
```

Comment:
What's the reason for using an actual Client class instead of a mock?

Reply:
Blob.download_to_file calls Client.download_blob_to_file, so we need actual Client class or we get an AttributeError

Reply:
Fair enough. Then I'd suggest using a `_make_one` factory for the `Client` instead of adding an import and a constructor  in every method that uses it.